src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -106,12 +112,22 @@ export default function ActionsPanel(): JSX.Element {
     string | undefined,
     SetSelectedAction,
   ] = useState<string>();
-  // First element in inLineErrors array corresponds to the get actions for
-  // application error. Second element corresponds to execute action error.
-  const [inlineErrors, setInlineErrors] = useState<(string | null)[]>([
-    null,
-    null,
-  ]);
+  const [inlineErrors, setInlineErrors, hasError] = useInlineErrors({
```

Comment:
For consistency reasons, maybe we could name `hasError` as `hasInlineErrors` instead?

Reply:
Done.